packages/shorebird_cli/lib/src/commands/release_new/release_new_command.dart

```diff
@@ -0,0 +1,420 @@
+import 'dart:async';
+
+import 'package:mason_logger/mason_logger.dart';
+import 'package:meta/meta.dart';
+import 'package:scoped/scoped.dart';
+import 'package:shorebird_cli/src/code_push_client_wrapper.dart';
+import 'package:shorebird_cli/src/command.dart';
+import 'package:shorebird_cli/src/commands/release_new/aar_releaser.dart';
+import 'package:shorebird_cli/src/commands/release_new/android_releaser.dart';
+import 'package:shorebird_cli/src/commands/release_new/release_type.dart';
+import 'package:shorebird_cli/src/commands/release_new/releaser.dart';
+import 'package:shorebird_cli/src/config/config.dart';
+import 'package:shorebird_cli/src/extensions/arg_results.dart';
+import 'package:shorebird_cli/src/logger.dart';
+import 'package:shorebird_cli/src/platform/platform.dart';
+import 'package:shorebird_cli/src/shorebird_env.dart';
+import 'package:shorebird_cli/src/shorebird_flutter.dart';
+import 'package:shorebird_cli/src/third_party/flutter_tools/lib/flutter_tools.dart';
```

Comment:
same here about removing this import

Reply:
This is for the exit override that lets us validate `exit` calls in tests.